tests/envs/registration/test_env_spec.py

```diff
@@ -86,43 +84,6 @@ def test_env_spec_to_from_json(env_spec: EnvSpec):
     assert env_spec == recreated_env_spec
 
 
-def test_wrapped_env_entry_point():
```

Comment:
Why is this test removed?

Reply:
I moved the tests to `test_make.py` I believe